frog/imports/api/activities.js

```diff
@@ -118,7 +119,7 @@ export const addActivity = (
     });
   else
     insertActivityMongo({
-      _id: uuid(),
```

Comment:
Here, I would recommend keeping this line and returning the generated id. In the `if` branch above, we can return the id provided